bitsandbytes/backends/npu.py

```diff
@@ -69,7 +181,7 @@ def int8_linear_matmul(
         out: Optional[torch.Tensor] = None,
         dtype=torch.int32,
     ) -> torch.Tensor:
-        raise NotImplementedError
+        return Int8AB(A, B)
```

Comment:
Interesting and clever! While this does break the expected API as this isn't returning a Tensor (or performing any operations really), I can completely understand _why_ it is done this way. I think this will be OK for right now and we'll make the interface better in this regard later on.